spring-cloud-gcp-samples/spring-cloud-gcp-logging-sample/README.adoc

```diff
@@ -25,9 +25,17 @@ $ curl --header "x-cloud-trace-context: [Some-Trace-ID]" localhost:8080/log
 To see the logged lines, navigate to the https://console.cloud.google.com/logs/viewer[Logging section of the Cloud Console].
 Select the `Global` resource and look under the `spring.log` channel to see the logged text.
 
+=== Running on Google Cloud Platform
+
+Deploy to Google AppEngine:
+
+----
+$ mvn clean package appengine:deploy -Dapp.deploy.projectId=$PROJECT_ID -Dapp.deploy.version=$VERSION
```

Comment:
What's the purpose of `clean` here?
Also, why do we have to specify project and version? I thought those were optional.

Reply:
`clean` wipes out anything prebuilt in the `target` folder - more just to ensure you're really deploying what you think you're deploying. It's not entirely _necessary_ but doesn't hurt anything either.

The new 2.2.0 plugin requires both the project ID and version to be specified in the config or passed via `-Dapp.deploy.[projectId | version]` - you get errors without it. I figured it was easier to do this in the command line vs an instruction on setting it in the config.